app/views/includes/search-form.nunjucks

```diff
@@ -9,7 +9,7 @@
     <div class="form-item-wrapper">
       {% if errorMessage %}
         <h2>{{ searchErrorLabel }}</h2>
-        <p>{{ errorMessage }} </p>
+        <p>{{ errorMessage }}</p>
       {% elif gps.length === 0 %}
          <h2>We can't find a surgery using '{{search}}'</h2>
```

Comment:
This is not different to the error Message/Label population and so would be better populated in the middleware so it can be tested. This can be raised as an issue.